assets/scripts/dialogs/AboutDialog.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * About Streetmix (dialog box)
+ *
+ * Handles the "About" dialog box.
+ * Instantiates an instance of Dialog
+ * Exports nothing
+ *
+ */
+import React from 'react'
+import Dialog from './Dialog.jsx'
```

Comment:
There are somethimes importes without file extention. We should stick to one standard and if it works I used to paths without extention.